### PR TITLE
Add description for every INVALID_ARGUMENT in ByteStreamService.

### PR DESCRIPTION
**Pull request by abergmeier-dsfishlabs**
Without that debugging errors is a PITA.

### CODE REVIEW

File: src/main/java/build/buildfarm/server/ByteStreamService.java [hunk 1]
```diff
@@ -232,7 +234,8 @@ private void writeBlob(
         if (data == null) {
           digest = parseUploadBlobDigest(writeResourceName);
           if (digest == null) {
-            responseObserver.onError(new StatusException(Status.INVALID_ARGUMENT));
+            String description = "Could not extract digest off: " + writeResourceName;
```
Comment (werkt) [inline src/main/java/build/buildfarm/server/ByteStreamService.java]: `Could not parse digest of:`

File: src/main/java/build/buildfarm/server/ByteStreamService.java [hunk 2]
```diff
@@ -259,7 +263,8 @@ private void writeBlob(
           active_write_requests.remove(writeResourceName);
           Digest blobDigest = Digests.computeDigest(data);
           if (!blobDigest.equals(digest)) {
-            responseObserver.onError(new StatusException(Status.INVALID_ARGUMENT));
+            String description = String.format("Digest mismatch %s <-> %s", blobDigest.getHash(), digest.getHash());
```
Comment (werkt) [inline src/main/java/build/buildfarm/server/ByteStreamService.java]: Prefer `Digests.toString(blobDigest)` etc here - the equals comparison occurs between both hash and size, and we have the toString mechanism representing all other presentations of digests

**Comment by abergmeier-dsfishlabs** [comment]
Done.

**Comment by werkt** [comment]
The web editor, which is capable of cleaning up this merge, creates commits that have Author lines that the googlebot cla resolver cannot identify as me (with a signed CLA). I cannot commit directly to @abergmeier-dsfishlabs' branch via the commandline, so I need to close this to get the merge in place. That PR will also require a review.

**Comment by werkt** [comment]
Recheck cla

**Pull request merged**